nautobot/core/views/generic.py

```diff
@@ -255,6 +255,16 @@ def get(self, request):
                 for field_name, values in filter_params.items()
             ]
 
+            if request.GET:
+                factory_formset_params = convert_querydict_to_factory_formset_acceptable_querydict(
+                    request.GET, self.filterset
+                )
+                self.dynamic_filter_form = DynamicFilterFormSet(
```

Comment:
Why does this need to be stored on `self`, instead of just being a local variable?